spec/views/form_drafts/show_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'form_drafts/show.haml' do
+  before :each do
+    @draft = create :form_draft
+  end
+  it 'contains the name of the form draft' do
+    render
+    expect(rendered).to have_tag 'h1', with: { class: 'title' } do
+      with_text(/#{@draft.name}/)
```

Comment:
Have you checked each of the `with_text` matchers to make sure this regex syntax is actually necessary?